actions/01nn.json

```diff
@@ -0,0 +1,34 @@
+{
+    "date": "2020/04/13",
+    "sources": [
+        "https://gizmodo.com/tech-workers-speak-out-in-support-of-amazon-warehouse-s-1842839301",
+        "https://chuffed.org/project/help-amazon-warehouse-workers-exposed-to-covid-19",
+        "https://www.givedirectly.org/covid-19/us/",
+        "https://twitter.com/techspeaksout"
+    ],
+    "actions": [
+        "protest"
+    ],
+    "struggles": [
+        "ethics",
+        "working_conditions",
+        "unfair_labor_practices"
+    ],
+    "employment_types": [
+        "white_collar_workers"
+    ],
+    "description": "As a gesture of solidarity, tech workers have started tweeting with the hashtags #TechSolidarity, #AmazonStrike, #HuelgaAmazon to indicate support for striking Amazon workers. The Twitter page, #TechSpeaksOut, was reportedly designed to exert public pressure tech corporations. In addition to taking to Twitter, activists have started GoFundMes and mutual aid campaigns to benefit workers.",
```

Comment:
`The Twitter page, #TechSpeaksOut` should be `The Twitter handle @TechSpeaksOut`